.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -107,7 +108,9 @@ body:
     attributes:
       label: Screenshot
       description: Please always add a screenshot or (even better) a screen recording to help us understand the problem.
-      placeholder: Paste screenshot or screen recording here.
+      placeholder: Attach screenshot or screen recording here.
+    validations:
+      required: true
```

Comment:
There are cases where a screenshot or screen recording cannot be provided, so let's remove these 2 lines again.

```suggestion
```

Reply:
Can I make the validation required: false so that I will still be there and anyone can submit without any screenshots or recordings?